compiler-core/src/javascript/tests/lists.rs

```diff
@@ -99,3 +99,33 @@ fn go(xs) {
 "#,
     );
 }
+
+#[test]
+fn tight_empty_list() {
+    assert_js!(
+        r#"
+fn go(func) {
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  use _ <- func(1, 2)
+  case [] {
+    [] -> []
+    _ -> []
+  }
```

Comment:
Do you think this could be rewritten just using a variable with a very long name that would trigger the same error?
```gleam
fn go() {
  let aaaaaaaaaa...a = []
//              ^^^ This would have to be long enough to trigger the
//                  code generation bug    
} 
```
So the snapshot could be easier to review

Reply:
Sure! Should be done now.